fusor-ember-cli/app/routes/review/installation.js

```diff
@@ -3,7 +3,7 @@ import request from 'ic-ajax';
 
 export default Ember.Route.extend({
   model() {
-    return this.modelFor('deployment');
+    return this.store.queryRecord('deployment', {deployment_id: this.modelFor('deployment').get('id'), sync_openstack: true});
```

Comment:
@cfchase, I assume this line is the reason for modifying ApplicationAdapter. However, it seems it should still work without the modification to ApplicationAdapter, but I could be missing something.